diff -urN arx-libertatis-1.1.1/src/core/ArxGame.cpp arx-libertatis-1.1.1-patched/src/core/ArxGame.cpp
--- arx-libertatis-1.1.1/src/core/ArxGame.cpp	2013-07-17 13:16:42.000000000 +1100
+++ arx-libertatis-1.1.1-patched/src/core/ArxGame.cpp	2013-07-30 01:28:32.000000000 +1100
@@ -257,7 +257,13 @@
 		
 		RenderWindow::DisplayMode mode(config.video.resolution, config.video.bpp);
 		if(mode.resolution == Vec2i::ZERO) {
-			mode = GetWindow()->getDisplayModes().back();
+			//mode = GetWindow()->getDisplayModes().back();
+			// Use current screen resolution by default because sometimes max resolution
+			// is not detected properly and the game doesn't start
+			const SDL_VideoInfo * vid = SDL_GetVideoInfo();
+			mode.resolution.x = vid->current_w;
+			mode.resolution.y = vid->current_h;
+			mode.depth = vid->vfmt->BitsPerPixel;
 		}
 		
 		// Clamp to a sane resolution!
@@ -299,7 +305,11 @@
 	if(config.video.resolution == Vec2i::ZERO) {
 		
 		// Use the largest available resolution.
-		mode = modes.back();
+		// mode = modes.back();
+		const SDL_VideoInfo * vid = SDL_GetVideoInfo();
+		mode.resolution.x = vid->current_w;
+		mode.resolution.y = vid->current_h;
+		mode.depth = vid->vfmt->BitsPerPixel;
 		
 	} else {
 		
